src/pages/cart/index.tsx

```diff
@@ -112,6 +169,12 @@ export default function Cart() {
       }, 0);
   }
 
+  // 제품 삭제
+  function handleProductRemove(id: number) {
+    const updatedProducts = products.filter(product => product.id !== id);
+    setProducts(updatedProducts);
```

Comment:
💊기존 값을 가져와서 사용하는 경우에는 setState의 업데이터 함수를 사용하면 더 좋을 거 같아요~

```suggestion
    setProducts((prev) => prev.filter(product => product.id !== id));
```
아래 링크에서 `업데이터`로 검색하시면 관련 내용 나옵니다! 참고하시면 좋을 거 같아요
https://react-ko.dev/reference/react/useState

Reply:
하나 또 배웠어요! 수정 완료했습니다 감사합니다 👏🏻